.circleci/config.yml

```diff
@@ -251,7 +251,45 @@ jobs:
           name: Use our node script to upload i18n frontend strings file
           command: node i18n/upload.js 'frontend.pot' '/../../src/frontend/i18n/frontend.pot'
 
-  test-back:
+  test-back-mysql:
+    machine: true
+    working_directory: ~/fun
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - docker-debian-images-dev-{{ .Revision }}
+      - run:
+          name: Load images to docker engine
+          command: |
+            docker load < docker/images/dev/richie.tar
+      # ElasticSearch configuration
+      #
+      # We need to increase the VM max memory size, or else, ElasticSearch (ES)
+      # service won't bootstrap.
+      #
+      # Source:
+      # https://www.elastic.co/guide/en/elasticsearch/reference/current/docker.html#docker-cli-run-prod-mode
+      - run:
+          name: Increase VM max memory size for ES
+          command: |
+            sudo sysctl -w vm.max_map_count=262144
+            sudo sysctl vm.max_map_count
+      # Run back-end (Django) test suite
+      #
+      # Nota bene: to run the django test suite, we need to ensure that both
+      # MySQL and ElasticSearch services are up and ready. To achieve this,
+      # we wrap the pytest command execution with dockerize, a tiny tool
+      # installed in the development image. In our case, dockerize will wait
+      # up to one minute that both the database and elastisearch containers
+      # opened their expected tcp port (3306 and 9200 resp.).
```

Comment:
Should we remove this comment and its sibling below?

Reply:
So we keep it for the moment and will remove it when the CircleCI config is refactored to use bin scripts.